lib/ah/lograge/httparty/log_subscriber.rb

```diff
@@ -0,0 +1,45 @@
+module Ah
+  module Lograge
+    module HTTParty
+      class LogSubscriber < ActiveSupport::LogSubscriber
+        STRIP_SUFFIX = %r[.ahinternal.net]
+        PREFIX = 'http_'
+        THREAD_KEY = :ah_lograge_http_runtime
+
+        def self.add_runtime(hostname, duration)
+          Thread.current[THREAD_KEY] ||= Hash.new(0)
+          Thread.current[THREAD_KEY][hostname] += duration
+        end
+
+        def self.reset_runtime
+          http_runtime = Thread.current[THREAD_KEY]
+          Thread.current[THREAD_KEY] = Hash.new(0)
+          return {} unless http_runtime
+          http_runtime.keys.each do |hostname|
+            http_runtime[service_key(hostname)] = http_runtime[hostname].round(2)
```

Comment:
I'm curious, why this transformation is needed here? The data stored in this hash is only used when reporting to the log. What do you think about handling such logic in `log_process_action`?